earthfile2llb/converter.go

```diff
@@ -1143,6 +1181,23 @@ func (c *Converter) Import(ctx context.Context, importStr, as string, isGlobal,
 	return c.varCollection.Imports().Add(importStr, as, isGlobal, currentlyPrivileged, allowPrivilegedFlag)
 }
 
+// Cache handles a `CACHE` command in a Target.
+// It appends run options to the Converter which will mount a cache volume in each successive `RUN` command,
+// and configures the `Converter` to persist the cache in the image at the end of the target.
+func (c *Converter) Cache(ctx context.Context, path string) error {
```

Comment:
Also, should there be an error if `CACHE` is seen after a `LOCALLY`?

Reply:
That sounds like a good idea. Does that also cover [your question above](https://github.com/earthly/earthly/pull/1514#discussion_r780568982)?

Reply:
Implemented this error

Reply:
Yeah, it does.